src/ResourceManager/HDInsight/Commands.HDInsight.Test/ScenarioTests/DataLakeStoreScenarioTests.cs

```diff
@@ -10,11 +10,18 @@ public DataLakeStoreScenarioTests(Xunit.Abstractions.ITestOutputHelper output)
             ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
         }
 
-        [Fact]
+        [Fact (Skip = "Test requires fixing GraphAPI by upgrading to latest test framework")]
```

Comment:
On second thought, I think I can just remove the tests that are being skipped currently.

The New-AzureRmHDInsightCluster cmdlet follows the same code path for WASB and ADLS. The only difference between the tests is that the ADL-tests are converting ObjectId into ApplicationId by using GraphAPI.

After converting ObjectId into ApplicationId, the value is included in ClusterCreateParameters and then the SDK takes over by using the clusterCreateParameters to create the cluster. 
The functionality that uses ClusterCreateParameters to create the cluster, is already being tested in SDK.